templates/web/fixmystreet.com/header_logo.html

```diff
@@ -1,5 +1,5 @@
 [% IF c.req.uri.path == '/' %]
-    <a href="https://www.mysociety.org/about/anniversary" id="mysociety-badge" aria-label="Visit mySociety 20 years anniversary">
+    <a href="https://www.mysociety.org/" id="mysociety-badge" aria-label="A services from mySociety">
```

Comment:
This should be "A service from mySociety":
```suggestion
    <a href="https://www.mysociety.org/" id="mysociety-badge" aria-label="A service from mySociety">
```

Reply:
Thanks @davea I just pushed a commit